test/integration/targets/vault/runme.sh

```diff
@@ -2,6 +2,10 @@
 
 set -euvx
 
+PYTHON=${ANSIBLE_TEST_PYTHON_INTERPRETER:-python}
+PIP=$(dirname ${ANSIBLE_TEST_PYTHON_INTERPRETER:-python})/pip
+
+
```

Comment:
Handling of pip and virtualenv in tests has changed very recently. You will need to rebase your branch to have access to these changes.

Replace the 4 added lines above with the following:

```suggestion
source virtualenv.sh
```

Reply:
done. thanks.